### DISCUSSION TITLE
Connection to endpoint failed

### DISCUSSION THREAD

**Discussion by heinzkrebs** [SafeExamBrowser/seb-win-refactoring#546] [Q&A]
servas,

on one of my students computer SEB is not starting. the client log say:

Trying to connect to endpoint 'net.pipe://localhost/safeexambrowser/runtime/ea2347.......
Communication channel is opening...
Endpoint 'net.pipe://localhost .....' could not be found!

is there a problem with the SEB service? the system is running Windows 10 ..

ciao
  H.

**Comment by dbuechel** [comment]
There then seems to be a problem with the inter-process communication between SEB's application components. This might be caused by anti-virus or other related security software blocking the communication, or then there is a general configuration issue on that system which prevents the .NET WCF framework from working correctly.
